plugins/quota/api/req.go

```diff
@@ -0,0 +1,13 @@
+package quota
+
+// LimitReq represents REST API request to Limit Usage/objects of a directory
+type LimitReq struct {
+	Path        string `json:"path"`
+	UsageLimit  int    `json:"usage-limit,omitempty"`
```

Comment:
I understand that these names come from how these features were implemented. However, we have a chance to change these legacy names for better ones. How about `UsageLimitSize` and `UsageLimitInode`. The names are intuitive.

Reply:
felt that usage would intutively mean disk space usage..
I find UsageLimitSize / UsageLimitInode a little odd.
How About:
SizeUsageLimit
ObjectCountLimit

Reply:
True. My bad. That sounds better.